api/src/poem_backend/transactions.rs

```diff
@@ -263,15 +263,10 @@ impl TransactionsApi {
     fn list(&self, accept_type: &AcceptType, page: Page) -> BasicResultWith404<Vec<Transaction>> {
         let latest_ledger_info = self.context.get_latest_ledger_info_poem()?;
         let ledger_version = latest_ledger_info.version();
+
         let limit = page.limit()?;
         // TODO: https://github.com/aptos-labs/aptos-core/issues/2286
-        let last_page_start = if ledger_version > (limit as u64) {
-            ledger_version - (limit as u64)
```

Comment:
this was actually a bug, it should've been limit -1. It currently skips the most recent transaction (version==ledger_version)
I fixed this (in compute_start) and added unit test for it.